src/aselo-webchat.tsx

```diff
@@ -174,7 +176,7 @@ export const initWebchat = async () => {
     const { language } = payload.formData;
 
     // Here we collect caller language (from preEngagement select) and change UI language
-    changeLanguageWebChat(language);
+    changeLanguageWebChat(externalWebChatLanguage || language);
```

Comment:
We probably don't want to override this selection with the value from the attribute? If there is a language attribute on the script tag AND a pre-engagement language selection, we would want take the user selection in preference

Reply:
Good catch

Reply:
@stephenhand, so I should revert as it was; `changeLanguageWebChat(language)`?

Reply:
@stephenhand change has been effected.